### ISSUE TITLE
New matchers to compare easily a number with another one

### ISSUE THREAD

**Issue by baudoliver7** [llorllale/cactoos-matchers#235]
We notice many times that we need to compare  a number with another one. Currently, outside of static methods of Hamcrest (`greaterThan`, `lessThan`), the more easy way to do this is to use `Verifies` like this : 
```java
new Verifies(x -> x > 0)
```

It would be very useful to introduce some new matchers to make this kind of assertions more expressive :

- `IsGreaterThan`
- `IsGreaterThanOrEqualTo`
- `IsLessThan`
- `IsLessThanOrEqualTo`

**Comment by victornoel** [comment]
@baudoliver7 thx

**Comment by victornoel** [comment]
@0crat in

**Comment by andreoss** [comment]
@victornoel Why not `Matcher<Comparable<?>>`?

**Comment by victornoel** [comment]
@andreoss yes, that would make sense, or `Matcher<Comparable<? extends T>>` (or would that be `super`? I think both would work actually here ^^). Let's note that even though `Number` does not implement `Comparable`, all its subclasses do. But in cactoos, the `Number` implementation must be wrapped in `ComparableNumber` though (it's not yet implemented but will be part of https://github.com/yegor256/cactoos/pull/1587)

**Comment by victornoel** [comment]
@0crat assign @andreoss

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/llorllale/cactoos-matchers/issues/235#issuecomment-858361978))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#235](https://github.com/llorllale/cactoos-matchers/issues/235) is [in scope](http://www.zerocracy.com/policy.html#14) for 4mon
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @baudoliver7[/z](https://www.0crat.com/u/baudoliver7): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Allorllale%2Fcactoos-matchers%23235%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)
